src/app-layout/__tests__/multi-layout.test.tsx

```diff
@@ -41,7 +41,8 @@ async function renderAsync(jsx: React.ReactElement) {
   const firstLayout = createWrapper().find('[data-testid="first"]')!.findAppLayout()!;
   const secondLayout = createWrapper().find('[data-testid="second"]')!.findAppLayout()!;
   expect(findAllToolbars()).toHaveLength(1);
-  expect(findToolbar(secondLayout)).toBeTruthy();
+  expect(findToolbar(firstLayout)).toBeTruthy();
+  expect(findToolbar(secondLayout)).toBeFalsy();
```

Comment:
Note: when app layouts are nested `findToolbar(firstLayout)` returns true even if it is inside the secondLayout.